### PR TITLE
pr for #3

### PR DESCRIPTION
**Pull request by KorolevskiTigr2**
pr for #3

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+        (юго-восточная Месопотамия) регулярно вели записи хронометража как
+        неотъемлемую часть своих астрономических наблюдений. Арабские
+        изобретатели и инженеры, в частности, внесли улучшения в использование
+        водяных часов вплоть до средневековья. ‎‎‎‎ В 11 веке ‎‎китайские
+        изобретатели‎‎ и ‎‎инженеры изобрели‎‎ первые механические часы,
+        приводимые в движение ‎‎спусковым‎‎ механизмом.‎
+      </p>
+    </section>
+    <!-- <link rel="stylesheet" href="./src/styles.css"> -->
+    <style>
```
Comment (gnrlbzik) [inline index.html:68]: nit: this has to be moved out into `style.css` file, @KorolevskiTigr2

File: index.html [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+  5.1 add text html
+  5.2 add hr (add size and color(black))
+  5.4 write text about time
+  5.5 add border(solid)
+  5.6 add new section 
+question 
+1. how delete last ":" (section 1)? (+)
+
+
+-->
```
Comment (gnrlbzik) [inline index.html:161]: nit: this can probably be removed or moved in to description of this PR. @KorolevskiTigr2

File: index.html [hunk 3]
```diff
@@ -0,0 +1,161 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>My project</title>
+    <meta charset="UTF-8" />
+  </head>
+  <body>
+    <section class="number-1">
```
Comment (gnrlbzik) [inline index.html:8]: nit: we need to do some sort of design exercise, to finalize look and feel of this page. And finish up styling of this.

Comment (gnrlbzik) [reply index.html:8]: lets pick some UI designs and/or UI elements from https://dribbble.com/, and build one out in photoshop, you can also do something in https://www.figma.com/ and share it with me.

File: index.html (hunk 1 above)
Comment (KorolevskiTigr2) [reply index.html]: Ready

Comment (gnrlbzik) [reply index.html]: thank you

File: index.html (hunk 3 above)
Comment (gnrlbzik) [reply index.html:9]: you can post that screen shot here, for historical reference

Comment (gnrlbzik) [reply index.html:9]: and then lets digest it into components and make them into html/css code as part of this PR. So once this is done, we can move to making all of this functioning/interactive with JS in next PR.

Comment (gnrlbzik) [reply index.html:9]: first component we will work
CleanShot 2022-07-01 at 13 10 56

Comment (gnrlbzik) [reply index.html:9]: Second component
CleanShot 2022-07-01 at 13 12 00

Comment (gnrlbzik) [reply index.html:9]: third component: header
CleanShot 2022-07-01 at 13 12 29

Comment (gnrlbzik) [reply index.html:9]: fourth component: conversion shortcut buttons
CleanShot 2022-07-01 at 13 13 41

Comment (gnrlbzik) [reply index.html:9]: fifth component: custom time zone conversion
CleanShot 2022-07-01 at 13 14 47

Comment (gnrlbzik) [reply index.html:9]: Example of app chrome to get you started: https://codesandbox.io/s/nostalgic-bassi-ekj43y?file=/index.html

File: Detalis/Third_component/Header.html [hunk 4]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
```
Comment (gnrlbzik) [inline Detalis/Third_component/Header.html:1]: nit: this file and folder structure should be `components/header/header.html`

File: Detalis/Third_component/Header.css [hunk 5]
```diff
@@ -0,0 +1,35 @@
+body {
```
Comment (gnrlbzik) [inline Detalis/Third_component/Header.css:1]: nit: this file and folder structure should be `components/header/header.css`

File: Detalis/Second_component/Clock.html [hunk 6]
```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
```
Comment (gnrlbzik) [inline Detalis/Second_component/Clock.html:1]: nit: this file and folder structure should be `components/clock/clock.html`

File: Detalis/Second_component/Clock.css [hunk 7]
```diff
@@ -0,0 +1,25 @@
+body {
```
Comment (gnrlbzik) [inline Detalis/Second_component/Clock.css:1]: nit: this file and folder structure should be `components/clock/clock.css`

File: Detalis/Second_component/Clock.css [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+    font-size: 3em;
+    font-weight: 800;
+  }
+  .section > div::after{
+    content: ":"; 
+  }
+  .section > div:last-child:after{
+    content: "AM";
+    padding: 5%;
+  }
```
Comment (gnrlbzik) [inline Detalis/Second_component/Clock.css:25]: nit: needs new line

File: Detalis/Second_component/Clock.html [hunk 9]
```diff
@@ -0,0 +1,19 @@
...
+<body>
+    <div id="Block">
+        <section class="section">
+          <div class="hours">21</div>
+          <div class="minutes">22</div>
+          <div class="second">56</div>
+        </div>
+        </section>
+</body>
+</html>
```
Comment (gnrlbzik) [inline Detalis/Second_component/Clock.html:19]: nit: needs new line

File: Detalis/Third_component/Header.css [hunk 10]
```diff
@@ -0,0 +1,35 @@
...
+    background-color: grey;
+    border-radius: 15%;
+    border: solid black;
+  }
+
+  .Dash {
+    font-size: 3em;
+    font-weight: 5em;
+    line-height: 5px;
+  }
```
Comment (gnrlbzik) [inline Detalis/Third_component/Header.css:35]: nit: needs new line

File: Detalis/Third_component/Header.html [hunk 11]
```diff
@@ -0,0 +1,15 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="Header.css">
+    <title>Header></title>
+</head>
+<body>
+    <div id="Block">
+        <p class="Text">WORLD CLOCK</p>
+        <input type="button" class="Button" />
+</body>
+</html>
```
Comment (gnrlbzik) [inline Detalis/Third_component/Header.html:15]: nit: needs new line

File: style.css [hunk 12]
```diff
@@ -0,0 +1,61 @@
...
+  }
+  
+  #select_country{
+	position: relative;
+	top: -2em;
+  }
+  #select_region{
+	position: relative;
+	left: -9.5em;
+  }
```
Comment (gnrlbzik) [inline style.css:61]: nit: needs new line

Comment (gnrlbzik) [reply style.css:61]: CleanShot 2022-07-02 at 20 13 22

File: Detalis/Third_component/Header.html (hunk 4 above)
Comment (gnrlbzik) [reply Detalis/Third_component/Header.html]: not done

File: Detalis/Third_component/Header.css (hunk 5 above)
Comment (gnrlbzik) [reply Detalis/Third_component/Header.css]: not done

File: Detalis/Second_component/Clock.html (hunk 6 above)
Comment (gnrlbzik) [reply Detalis/Second_component/Clock.html]: not done

File: Detalis/Second_component/Clock.css (hunk 7 above)
Comment (gnrlbzik) [reply Detalis/Second_component/Clock.css]: not done

**Comment by gnrlbzik** [comment]
here is the structure of the files that you need to follow:

```
project_root/
  - components/
    - clock/
      - clock.css
      - clock.html
```

File: Detalis/Components_Clock/Clock.css [hunk 13]
```diff
@@ -0,0 +1,26 @@
...
+    font-weight: 800;
+  }
+  .section > div::after{
+    content: ":"; 
+  }
+  .section > div:last-child:after{
+    content: "AM";
+    padding: 5%;
+  }
+
```
Comment (gnrlbzik) [inline Detalis/Components_Clock/Clock.css:26]: nit: still need to fix new line issue

File: Detalis/Components_Header/Header.css [hunk 14]
```diff
@@ -0,0 +1,40 @@
...
+    border-radius: 15%;
+    border: solid black;
+  }
+
+  .Dash {
+    font-size: 3em;
+    font-weight: 5em;
+    line-height: 5px;
+  }
+
```
Comment (gnrlbzik) [inline Detalis/Components_Header/Header.css:40]: nit: still need to fix new line issue

File: Detalis/Components_Header/Header.html [hunk 15]
```diff
@@ -0,0 +1,15 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="Header.css">
+    <title>Header></title>
+</head>
+<body>
+    <div id="Block">
```
Comment (gnrlbzik) [inline Detalis/Components_Header/Header.html:11]: nit: you missed closing div tag `</div>`

File: Detalis/Components_Header/Header.html [hunk 16]
```diff
@@ -0,0 +1,15 @@
...
+        <p class="Text">WORLD CLOCK</p>
+        <input type="button" class="Button" />
```
Comment (gnrlbzik) [inline Detalis/Components_Header/Header.html:13]:
```suggestion
    <div id="Block">
        <p class="Text">WORLD CLOCK</p>
        <input type="button" class="Button" />
     </div>
```

File: style.css [hunk 17]
```diff
@@ -0,0 +1,62 @@
...
+
```
Comment (gnrlbzik) [inline style.css:62]: nit: same thing, no newline at end of file https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log

File: index.html (hunk 3 above)
Comment (gnrlbzik) [reply index.html:9]: CleanShot 2022-07-03 at 15 15 05

Comment (gnrlbzik) [reply index.html:9]: https://codesandbox.io/s/nostalgic-bassi-ekj43y?file=/index.html

**Comment by gnrlbzik** [comment]
request (blocking): please work on improving your commit messages @KorolevskiTigr2

**Comment by gnrlbzik** [comment]
fix: `Project_root/Components/Header/Header.html` => `src/components/header/header.html`, use same format for rest of the files

File: Project_root/Components/Clock/Clock.css [hunk 18]
```diff
@@ -0,0 +1,26 @@
+body {
+    background-color: orange;
+  }
+  #Block {
```
Comment (gnrlbzik) [inline Project_root/Components/Clock/Clock.css:4]: fix: no capital letters please
```suggestion
  #block {
```

File: Project_root/Components/Clock/Clock.html [hunk 19]
```diff
@@ -0,0 +1,19 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="Clock.css">
+    <title>Clock</title>
+</head>
+<body>
+    <div id="Block">
```
Comment (gnrlbzik) [inline Project_root/Components/Clock/Clock.html:11]: fix: update id
```suggestion
    <div id="block">
```

**Comment by gnrlbzik** [comment]
CleanShot 2022-07-08 at 12 40 00
 just needs to be 'Components'

**Comment by gnrlbzik** [comment]
CleanShot 2022-07-08 at 12 41 11

nit: no need to use "commit message: " pre fix

**Review by gnrlbzik — APPROVED** [review]
lets merge as is, and focus on separate items in separate prs

**Pull request merged**